share/models/change.py

```diff
@@ -177,6 +177,8 @@ def _create(self, save=True):
             except IntegrityError as e:
                 from share.disambiguation import disambiguate
                 logger.info('Handling unique violation error %r', e)
+                if self.target_type.model == 'identifier':
```

Comment:
Having a unique violation on identifier should be an acceptable case. It is often unavoidable when handling large amounts of data. Anytime a `MakeJsonPatches` job is run while there are pending changes this block could be hit.